data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/utils/ExhaustiveCracCreation.java

```diff
@@ -23,10 +23,10 @@
  * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
  */
 public final class ExhaustiveCracCreation {
-    private static final String PREVENTIVE_INSTANT_ID = "preventive";
-    private static final String OUTAGE_INSTANT_ID = "outage";
-    private static final String AUTO_INSTANT_ID = "auto";
-    private static final String CURATIVE_INSTANT_ID = "curative";
+    public static final String PREVENTIVE_INSTANT_ID = "preventive";
+    public static final String OUTAGE_INSTANT_ID = "outage";
+    public static final String AUTO_INSTANT_ID = "auto";
+    public static final String CURATIVE_INSTANT_ID = "curative";
```

Comment:
I don't love these changes ; these IDs are supposed to be used with unit tests that use the CRAC created by this class only